### PR TITLE
Enable C# emit tests on Linux CoreClr

### PR DESCRIPTION
**Pull request by jaredpar**

### CODE REVIEW

**Comment by jaredpar** [comment]
CC @dotnet/roslyn-compiler for review

File: src/Compilers/CSharp/Test/Emit/Emit/DynamicAnalysis/DynamicInstrumentationTests.cs [hunk 1]
```diff
@@ -1754,7 +1755,8 @@ File 1
             verifier.VerifyDiagnostics();
         }
 
-        [Fact]
+        // Disabling to track down deadlock
```
Comment (333fred) [inline src/Compilers/CSharp/Test/Emit/Emit/DynamicAnalysis/DynamicInstrumentationTests.cs]: Do we have a bug tracking this?

File: src/Compilers/CSharp/Test/Emit/PDB/PDBTests.cs [hunk 2]
```diff
@@ -2234,55 +2237,7 @@ .maxstack  2
 ", sequencePoints: "Program+<Test>d__0.MoveNext", source: source);
         }
 
-        [Fact]
-        [WorkItem(12564, "https://github.com/dotnet/roslyn/issues/12564")]
-        public void ConditionalInSimpleMethod()
```
Comment (333fred) [inline src/Compilers/CSharp/Test/Emit/PDB/PDBTests.cs]: Why delete this test?

**Comment by jaredpar** [comment]
The tests are hanging on Ubuntu. In digging into that I found a deeper issue I need to solve before I can merge this. Marking as blocked while I look into that.

File: src/Compilers/CSharp/Test/Emit/Emit/DynamicAnalysis/DynamicInstrumentationTests.cs (hunk 1 above)
Comment (jaredpar) [reply src/Compilers/CSharp/Test/Emit/Emit/DynamicAnalysis/DynamicInstrumentationTests.cs]: This is a mistake. Thanks for catching. Will undo.

File: src/Compilers/CSharp/Test/Emit/PDB/PDBTests.cs (hunk 2 above)
Comment (jaredpar) [reply src/Compilers/CSharp/Test/Emit/PDB/PDBTests.cs]: That's a mistake. Thank you for catching.

---
In reply to: [211443158](https://github.com/dotnet/roslyn/pull/29381#discussion_r211443158) [](ancestors = 211443158)

**Pull request merged**